### DISCUSSION TITLE
Deep link format

### DISCUSSION THREAD

**Discussion by kazbeksultanov** [WalletConnect/walletconnect-monorepo#948] [Developer Support]
Hi, I just want to clarify.

In wallet connect account while registering the app, in field Deep Link we have to write "myapp", "myapp:" or "myapp://"?
Could someone clarify this moment? 

![Снимок экрана 2022-04-14 в 20 03 49](https://user-images.githubusercontent.com/17141647/163407437-d032f02a-5498-4936-8774-f5b205400fec.png)

Thanks

**Comment by chadyj** [comment]
`myapp:`

**Reply by kazbeksultanov** [reply]
Thanks

**Reply by kazbeksultanov** [reply]
I have to submit through the cloud registry and wait until it processed? There are no other faster ways to change deep link?

**Reply by chadyj** [reply]
Reviews are pretty fast. In fact, it has already been published :)
